### PR TITLE
[Finishes #156299993] Crud tests remove excessive spacing

### PR DESCRIPTION
**Pull request by simiyu1**
Removed excessive spacing by adding *.pyc in .gitignore .

### CODE REVIEW

File: app/test/test_cruds.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+class booksTestCase(unittest.TestCase):
+    """In this class we shall be testing the API endpoints"""
+
+    def setUp(self):
+        """Define test variables and initialize some list data for the app."""
+        self.app = create_app(config_name="testing")
+        self.client = self.app.test_client
+        self.book = {'ISBN': '475', 'title':'Night train to Istanbull','author':'Penny Mice',
+                      'edition':'1','publisher':'Pearson','copies':'54'}
+
```
Comment (geeeh) [inline app/test/test_cruds.py]: Why have two lines between functions?

File: app/v1/__init__.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
+
+from flask_api import FlaskAPI
+from flask import request, jsonify, abort
+
+
+# local import configs and routes(route file merged here)
+from instance.config import app_config
+
```
Comment (geeeh) [inline app/v1/__init__.py]: Why all this space here?

File: app/v1/__init__.py [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+                    'author': booklist.author,
+                    'edition': booklist.edition,
+                    'publisher': booklist.publisher,
+                    'copies': booklist.copies
+                }
+                results.append(obj)
+            response = jsonify(results)
+            response.status_code = 200
+            return response
+
```
Comment (geeeh) [inline app/v1/__init__.py]: Functions are separated by single empty lines and classes by two empty lines.

File: app/test/test_cruds.py (hunk 1 above)
Comment (simiyu1) [reply app/test/test_cruds.py]: Was trying to avoid an extremely long line of code...

File: app/v1/__init__.py (hunk 2 above)
Comment (simiyu1) [reply app/v1/__init__.py]: Let me Google how to use your code to remove extra spacing... seems like I did not do it properly.

File: app/v1/__init__.py (hunk 3 above)
Comment (simiyu1) [reply app/v1/__init__.py]: Fixing this. Thank you

**Comment by simiyu1** [comment]
Fixing this too sir..

**Pull request merged**